<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-G0D562V" name="scripts.DCRM.DCRMScenarios_Test" tests="3" failures="0" timestamp="2020-05-30T19:32:09 IST" time="29.325" errors="1">
  <testcase name="myCase" time="14.204" classname="scripts.DCRM.DCRMScenarios_Test"/>
  <system-out/>
  <testcase name="launchReport" time="7.785" classname="scripts.DCRM.DCRMScenarios_Test">
    <error type="org.apache.commons.mail.EmailException" message="Cannot attach file &quot;./reports/SummaryReport_30-May-2020 19-31/TestSummary.html&quot;">
      <![CDATA[org.apache.commons.mail.EmailException: Cannot attach file "./reports/SummaryReport_30-May-2020 19-31/TestSummary.html"
at org.apache.commons.mail.MultiPartEmail.attach(MultiPartEmail.java:304)
at utilities.ExtentReportHtml.sendMail(ExtentReportHtml.java:161)
at utilities.ExtentReportHtml.launchReport(ExtentReportHtml.java:170)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:64)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:364)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:318)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:351)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.io.IOException: "./reports/SummaryReport_30-May-2020 19-31/TestSummary.html" does not exist
at org.apache.commons.mail.MultiPartEmail.attach(MultiPartEmail.java:292)
... 21 more
]]>
    </error>
  </testcase> <!-- launchReport -->
  <system-out/>
  <testcase name="myCase" time="7.336" classname="scripts.DCRM.DCRMScenarios_Test"/>
  <system-out/>
</testsuite> <!-- scripts.DCRM.DCRMScenarios_Test -->
